private/bufpkg/bufpolicy/policy_config.go

```diff
@@ -390,6 +397,68 @@ func marshalPolicyConfigAsJSON(policyConfig PolicyConfig) ([]byte, error) {
 	return data, nil
 }
 
+func unmarshalJSONPolicyConfig(registry string, data []byte) (PolicyConfig, error) {
```

Comment:
This works, thanks! I wouldn't have a problem either if this returned the `*policyv1beta1.PolicyConfig` type instead.